packages/kit/src/runtime/client/fetcher.js

```diff
@@ -67,6 +67,22 @@ if (DEV) {
 
 const cache = new Map();
 
+/**
+ * @param {string} text
+ * @returns {ArrayBufferLike}
+ */
+function b64_decode(text) {
+	const d = atob(text);
+
+	const u8 = new Uint8Array(d.length);
+
+	for (let i = 0; i < d.length; i++) {
+		u8[i] = d.charCodeAt(i);
+	}
+
+	return u8.buffer;
+}
+
```

Comment:
`Uint8Array.from` could be optimized under the hood?

I think this is equivalent `Uint8Array.from(atob(body), (c) => c.charCodeAt(0))`

Reply:
I did some testing on node v20.5.1 Ryzen 9 5950X 32GB

`Uint8Array.from(atob(text), (c) => c.charCodeAt(0))` takes ~74ms to decode 1MB (out) of data
```js
const d = atob(text);
return Uint8Array.from({length: d.length}, (_, i) => d.charCodeAt(i));
```
takes ~50ms

And using the for loop, ~17ms

`Uint8Array.from(atob(text), (c) => c.charCodeAt(0))` is the only one with any major gc